### PR TITLE
fix: enable processing:software summaries field, remove requirement from historical imagery collection

### PR DESCRIPTION
**Pull request by amfage**

### CODE REVIEW

File: fields/fields.json [hunk 1]
```diff
@@ -553,7 +553,7 @@
     "processing:software": {
       "label": "Software",
       "format": "Software",
-      "summary": false
+      "summary": "v"
```
Comment (l0b0) [inline fields/fields.json:556]: Where is this value documented? There doesn't seem to be a mention in this file or the readme. I only vaguely remember fields.json being in some existing format.

Also, the comment at the top of this file says it's generated. If that's the case, why are we version controlling it? Shouldn't it be generated only when we need it, for example to copy to stac.linz.govt.nz?

Comment (amfage) [reply fields/fields.json]: Megan explained it to me, I guess we can't override this, so I've reverted the change.

**Review by MDavidson17 — APPROVED** [review]
:+1:

**Comment by amfage** [comment]
Closed as needs more internal discussion about when to require fields.

**Pull request closed without merging**